packages/uploadthing/test/adapters.test.ts

```diff
@@ -49,8 +49,6 @@ describe("adapters:h3", async () => {
         middlewareMock(opts);
         expectTypeOf<{
           event: H3Event;
-          req: undefined;
-          res: undefined;
```

Comment:
this could technically be seen as a breaking change but cmon...

Reply:
undefined -> unknown ? cmon 😅